css/_filmstrip.scss

```diff
@@ -48,7 +48,6 @@
     &__videos {
         @extend %align-right;
         position:relative;
-        height:196px;
```

Comment:
Not sure why this was here, but it was messing with my new invite button UI, so I removed it. Tested in horizontal and vertical filmstrip mode and it didn't seem to have any consequence.

Reply:
Sounds good. We shouldn't be afraid to kill off CSS because we can likely fix it easy enough.